src/Dev/Deprecation.php

```diff
@@ -360,6 +434,24 @@ public static function notice($atVersion, $string = '', $scope = Deprecation::SC
         }
     }
 
+    /**
+     * Shorthand method to create a suppressed notice for something with no immediate replacement.
+     * If $string is empty, then a standardised message will be used, which is:
+     * Will be removed without equivalent functionality to replace it.
+     */
+    public static function noticeWithNoReplacment(
+        string $atVersion,
+        string $string = '',
+        int $scope = Deprecation::SCOPE_METHOD
+    ): void {
+        if ($string === '') {
+            $string = 'Will be removed without equivalent functionality to replace it.';
+        }
+        Deprecation::withSuppressedNotice(
+            fn() => Deprecation::notice($atVersion, $string, $scope)
```

Comment:
```suggestion
        string $message = '',
        int $scope = Deprecation::SCOPE_METHOD
    ): void {
        if ($message === '') {
            $message = 'Will be removed without equivalent functionality to replace it.';
        }
        Deprecation::withSuppressedNotice(
            fn() => Deprecation::notice($atVersion, $message, $scope)
```
"string" is not a useful variable name - what does the string represent? What is it used for?

Reply:
It's the param name, I'd prefer $message though notice() already has that and I want to remain consistent

Reply:
> I'd prefer $message though notice() already has that and I want to remain consistent

I don't personally think we should be consistent when the thing we're being consistent with is objectively bad, but it doesn't matter much so I won't block merging for this.

Reply:
Updated